exa: add --header, --git and --git-ignore examples (#5749)
tldr-pages/tldr: pages/common/exa.md

- List files sorted by modification date (oldest first):

`exa --long --sort={{modified}}`

> - Long list of files with header, icons, git status:

Reviewer: Perhaps this will read better?
```suggestion
- List files with headers, icons, and git status:
```

Reviewer 2: Or this?
```suggestion
- List files with their headers, icons, and git statuses:
```

> `exa --long --header --icons --git`

Reviewer 3: :thinking: What if we are `--git-ignore` in the example? or we can add a disclaimer in the example description like 
```
(use `--git-ignore` to ...)
```

Reviewer 3: @bl-ue you :+1: which option ?

Reviewer 3: the help says:
```
--git-ignore               ignore files mentioned in '.gitignore'
```
basically you don't see files or folders ignored by git in the output list, e.g. `node_modules/`

Reviewer: Maybe we can add a whole new example, too.  But I don't even think it matters that much—this is @ZabejAga's PR, and they can decide which option (if any) they'd like.  @ZabejAga?
